src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -367,6 +355,14 @@ protected virtual void Initialize()
             }
         }
 
+        private void TraceFileChangeRestart(string changeType, string path, bool isShutdown)
+        {
+            TraceWriter.Info(string.Format(CultureInfo.InvariantCulture, "File change of type '{0}' detected for '{1}'", changeType, path));
```

Comment:
I'm keeping the existing logging logic we have in the host today (https://github.com/Azure/azure-webjobs-sdk-script/blob/dev/src/WebJobs.Script/Host/ScriptHost.cs#L317-L318).

I agree that this should probably only come from the primary host, though.